components/Stripe/SubscriptionModal.tsx

```
@@ -0,0 +1,230 @@
+  DialogTrigger,
+} from "../ui/dialog";
+import { Loader2, Plus, CreditCard, Shield, Zap, Crown } from "lucide-react";
+import { Button } from "../ui/button";
+import { toast } from "sonner";
+import {
+  onGetStripeClientSecret,
+  updateSubscription,
+} from "@/actions/stripe.actions";
+import Stripe from "stripe";
+
+const SubscriptionModal = ({ user }: SubscriptionModal) => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'SubscriptionModal'. Consider to delete it or rename it.

'SubscriptionModal' is defined here:


(lint/suspicious/noRedeclare)